packages/contracts/src/dollar/libraries/LibStaking.sol

```diff
@@ -267,8 +267,8 @@ library LibStaking {
         // deposit new shares
         LibChef.deposit(msg.sender, _sharesAmount, _id);
         // calculate end locking period block number
-        // 1 week = 45361 blocks = 2371753*7/366
-        // n = (block + duration * 45361)
+        // 1 week = 49930 blocks
+        // n = (block number + duration * 49930)
         stake.endBlock = block.number + _weeks * ss.blockCountInAWeek;
```

Comment:
i think we should improve ALL naked math with

```solidity
unchecked { }
```

i think its best practice for upcoming updates

Reply:
@molecula451 the best would be to open a new issue, describe why it is needed and we can address this in a separate pull request.

Reply:
yes just saying